server/tests/test-config.json

```diff
@@ -90,7 +90,7 @@
     },
     "SqlSettings": {
         "DriverName": "mysql",
-        "DataSource": "mmuser:mostest@tcp(localhost:3306)/mattermost_test?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s\u0026maxAllowedPacket=4194304",
+        "DataSource": "mmuser:mostest@tcp(localhost:3306)/mattermost_test?charset=utf8mb4\u0026readTimeout=30s\u0026writeTimeout=30s\u0026maxAllowedPacket=4194304",
```

Comment:
Just going with the default charset for MySQL 8. This change is already documented. Don't think there's anything new to document.

Reply:
What I am referring to is that in pretty much all of our database related documents (e.g. https://docs.mattermost.com/configure/configuation-in-mattermost-database.html) we have the former version. If it's not a problem then fine to leave it otherwise I'd suggest to plan a documentation review.

Reply:
It's not a problem of course. But I think it makes sense to do a full review and change all of those strings for v8.

Reply:
/cc @cwarnermm for visibility